src/client/game.cpp

```diff
@@ -539,14 +592,17 @@ class GameGlobalShaderConstantSetterFactory : public IShaderConstantSetterFactor
 	bool *m_force_fog_off;
 	f32 *m_fog_range;
 	Client *m_client;
+	Camera *m_camera;
 	std::vector<GameGlobalShaderConstantSetter *> created_nosky;
+	std::vector<GameGlobalShaderConstantSetter *> created_camera;
 public:
 	GameGlobalShaderConstantSetterFactory(bool *force_fog_off,
 			f32 *fog_range, Client *client) :
 		m_sky(NULL),
 		m_force_fog_off(force_fog_off),
 		m_fog_range(fog_range),
-		m_client(client)
+		m_client(client),
+		m_camera(NULL)
```

Comment:
`nullptr`?

Reply:
It was in the original PR, I plan to do a big `sed -i 's/NULL/nullptr/g'` in the code at some point so it's not a big deal. I'll fix it tho

Reply:
Ah, ok. Similar PR: #6759

Reply:
I saw that one, I'll do another PR like this after 5.0.0